d2/src/main/java/com/linkedin/d2/balancer/util/hashing/MPConsistentHashRing.java

```diff
@@ -53,6 +53,16 @@ public class MPConsistentHashRing<T> implements Ring<T>
   public static final int DEFAULT_POINTS_PER_HOST = 1;
 
   private static final Logger LOG = LoggerFactory.getLogger(ConsistentHashRing.class);
+  static {
+    try {
+      LongHashFunction.class.getMethod("xx_r39", long.class);
+    } catch (NoSuchMethodException ex) {
+      // SI-36497
```

Comment:
this is OSS code, may not be proper to put internal JIRA ticket.

Reply:
That thought did occur to me but then I remembered seeing a lot of such internal ticket and doc links in gRPC-java code :) Anyhow, removed now.

Reply:
grpc-infra is not OSS?

Reply:
Was referring to `grpc-java`, but I may be mistaken and we don't need to repeat their mistakes anyway. Point taken.